showmore_dyn/showmore_dyn.php

```diff
@@ -0,0 +1,34 @@
+<?php
+/**
+ * Name: Showmore Dynamic
+ * Description: Dynamically limits height of posts
+ * Version: 1.0
+ * Author: Christian Wiwie
+ *
+ */
+
+use Friendica\App;
+use Friendica\Core\Hook;
+use Friendica\DI;
+
+function showmore_dyn_install() {
+	Hook::register('head'  , __FILE__, 'showmore_dyn_head');
+        Hook::register('footer', __FILE__, 'showmore_dyn_footer');
+}
+
+function showmore_dyn_uninstall()
```

Comment:
If your `uninstall` method only has `Hook::unregister` calls you can skip it entirely, all hooks are unregistered by default when uninstalling any addon. It is a recent behavior change which explains that not all the addons are following it.